app/assets/javascripts/wizard/addon/components/homepage-preview.js

```diff
@@ -4,7 +4,7 @@ import {
   darkLightDiff,
 } from "wizard/lib/preview";
 
-export default createPreviewComponent(659, 320, {
+export default createPreviewComponent(342, 322, {
```

Comment:
Haven't tried it locally - why such big dimensions change? 😃

Reply:
The new wizard has reduced the height of the content box which resulted in small preview window.

Old:

<img width="841" alt="Screen Shot 2022-07-25 at 12 30 28" src="https://user-images.githubusercontent.com/5732281/180717355-3845241e-0478-4437-8e01-44d00e4f91ee.png">

New:

<img width="866" alt="Screen Shot 2022-07-25 at 12 30 58" src="https://user-images.githubusercontent.com/5732281/180717383-7efaab9b-d4f6-4548-a0f1-655e99ed0d7a.png">